.github/workflows/ci.yml

```diff
@@ -171,7 +171,6 @@ jobs:
             'eslint-plugin-internal',
             'parser',
             'rule-tester',
-            'rule-schema-to-typescript-types',
```

Comment:
This project doesn't have a `test` target anymore. It didn't contain any spec files, so the `jest.config.js` file was removed.